### PR TITLE
Engg

### PR DESCRIPTION
**Pull request by sananand007**
Added Frontend changes on the home page and redirects links

### CODE REVIEW

File: service_flask.py [hunk 1]
```diff
@@ -17,6 +18,16 @@ def welcome():
...
     return render_template("welcome_page.html")
 
+@app.route("/getTime", methods=["GET"])
+def getTime():
+    """
+    Print the time on the welcome/main page
+    """
+    now = datetime.date.today()
+    #formatted_now = now.strftime("%A, %d %B, %Y at %X")
+    content = f"Thanks for visiting, Time now {now}"
```
Comment (weihesdlegend) [inline service_flask.py]: nice f string!

File: service_flask.py [hunk 2]
```diff
@@ -17,6 +18,16 @@ def welcome():
     # return "Welcome to use the auto-complete system!"
     return render_template("welcome_page.html")
 
+@app.route("/getTime", methods=["GET"])
+def getTime():
+    """
+    Print the time on the welcome/main page
+    """
+    now = datetime.date.today()
+    #formatted_now = now.strftime("%A, %d %B, %Y at %X")
```
Comment (weihesdlegend) [inline service_flask.py]: remove commented code

File: service_flask.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 from src.Server import Server
 from flask import Flask, url_for, request, render_template
+import time, datetime
```
Comment (weihesdlegend) [inline service_flask.py]: is time module used?

File: service_flask.py (hunk 2 above)
Comment (sananand007) [reply service_flask.py]: Removed all commented code

File: service_flask.py (hunk 3 above)
Comment (sananand007) [reply service_flask.py]: time is not used, taken that out. Datetime still there

**Comment by weihesdlegend** [comment]
closing for now

**Pull request closed without merging**